packages/cli/__tests__/daemon/__integration__/disable/cli.daemon.disable.integration.ts

```diff
@@ -44,32 +43,23 @@ describe("daemon disable", () => {
         const sysInfo: ISystemInfo = ProcessUtils.getBasicSystemInfo();
 
         // form our tgz file name
-        let tgzFileName = "zowe-";
         switch (sysInfo.platform) {
-            case "darwin": {
-                tgzFileName += "macos.tgz";
-                exePath = "zowe";
-                break;
-            }
+            case "darwin":
             case "linux": {
-                tgzFileName += "linux.tgz";
                 exePath = "zowe";
                 break;
             }
             case "win32": {
-                tgzFileName += "windows.tgz";
                 exePath = "zowe.exe";
                 break;
             }
             default: {
-                tgzFileName += "unknownOs.tgz";
                 exePath = "exeForUnknownOs";
                 throw "cli.daemon.disable.integration.test.ts: beforeAll: " + sysInfo.platform + " is not a known OS.";
```

Comment:
Not sure if it's desirable to hardcode the filename in the tests. When the file is renamed like in this PR it will become out-of-date unless we remember to change it. Feel free to disregard this comment since this is just an error message where the filename doesn't really matter 🙂

If we want to change this, I'd suggest either using the Node.js built-in variable `__filename` or just leaving the filename out if we trust Jest to show us which test file threw an error.

Reply:
The above comment also applies to line 77 of "cli.daemon.enable.integration.ts", which I'm not able to comment on because its not part of the diff.

Reply:
Good thought on the use of __filename.